pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -164,18 +164,8 @@ func (s *Service) HandleInbound(msg *service.DIDCommMsg) (string, error) {
 
 	internalMsg := &message{Msg: msg, ThreadID: thID, NextStateName: next.Name(), ConnRecord: connRecord}
 
-	// trigger action event based on message type for inbound messages
-	if canTriggerActionEvents(msg.Header.Type) {
-		if err = s.sendActionEvent(internalMsg, aEvent); err != nil {
-			return "", fmt.Errorf("handle inbound : %w", err)
-		}
-
-		return connRecord.ConnectionID, nil
-	}
-
-	// if no action event is triggered, continue the execution
 	go func(msg *message) {
-		if err = s.handle(msg); err != nil {
+		if err = s.handle(msg, aEvent); err != nil {
```

Comment:
any reason aEvent isn't an argument like msg?

Reply:
missed this. I'll update it.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/0acce2085bbdff34cadccb9d86d74957ba12b028..c3fbb046885a794db280ed57a43d363cda1b2742#diff-f761c91e753eba1c42a68076df06ce12R171).